docs/install/README.md

```diff
@@ -7,30 +7,54 @@ description: There are several different ways you can install and interact with
 
 IPFS is a collection of protocols, packages, and specifications that allow computers to send and receive data. Because of this, users can interact with and use IPFS in many different ways. A developer building network applications will use a different set of tools to interact with IPFS than someone who wants to store files on IPFS. Pick the one that best suits what you're here to do.
 
-## Publish files with a pinning service
+Looking for an easy and opinionated way to get started with IPFS [Mainnet](../concepts/glossary.md#mainnet)? Try any of the options listed below:
 
-Do you want to quickly and easily publish content with IPFS without complex tools? See the [Publish with IPFS quickstart](../quickstart/publish.md), where you'll learn how to use pinning services to pin files to the IPFS network.
+## User-Friendly Options
 
-## IPFS Desktop
+### IPFS Desktop
 
 Anyone can use IPFS to store files in a _decentralized_ way. The easiest way to get up and running is by installing the IPFS Desktop application. This app has Kubo built-in and lets you interact with the network through a simple user-interface. [Check it out →](./ipfs-desktop.md)
 
-## IPFS Kubo Node and CLI
+### IPFS Companion
 
-Want to build decentralized applications and store your application data on IPFS? You'll likely want to install the command-line version of IPFS. There's no GUI to deal with, just raw input and output through your terminal. [Find out more →](./command-line.md)
+If your browser doesn't support IPFS yet, you can install an IPFS companion extension that will let you view decentralized web content! [Learn more →](./ipfs-companion.md)
+
+### Publish files with a pinning service
+
+Do you want to quickly and easily publish content with IPFS without complex tools? See the [Publish with IPFS quickstart](../quickstart/publish.md), where you'll learn how to use third-party pinning services to pin and provide files to the IPFS network.
+
+## Command-Line & Infrastructure Tools
```

Comment:
Any reason Kubo shouldn't move to "Software Development" (line 44)? That would allow this section to be, simply, "Infrastructure Tools".

Reply:
Kubo is a full opinionated Node node, not a library.
But I've renamed this to "Infrastructure Tools" because Kubo is what people use in infrastructure.